source/base/parameter_handler.cc

```diff
@@ -516,19 +633,45 @@ namespace Patterns
 
 
 
-  std::string List::description () const
+  std::string List::description (const OutputStyle style) const
   {
-    std::ostringstream description;
+    switch (style)
+      {
+      case Machine:
+      {
+        std::ostringstream description;
 
-    description << description_init
-                << " list of <" << pattern->description() << ">"
-                << " of length " << min_elements << "..." << max_elements
-                << " (inclusive)";
-    if (separator != ",")
-      description << " separated by <" << separator << ">";
-    description << "]";
+        description << description_init
+                    << " list of <" << pattern->description(style) << ">"
+                    << " of length " << min_elements << "..." << max_elements
+                    << " (inclusive)";
+        if (separator != ",")
+          description << " separated by <" << separator << ">";
+        description << "]";
 
-    return description.str();
+        return description.str();
+      }
+      case Text:
+      case LaTeX:
+      {
+        std::ostringstream description;
+
+        description << "a list of "
+                    << min_elements << " to " << max_elements
+                    << " elements ";
+        if (separator != ",")
+          description << "separated by <" << separator << "> ";
+        description  << "where each element is "
+                     << pattern->description(style);
```

Comment:
Yes, it occurred to me later down in the patch (when I got to lists etc) why you were not capitalizing the first word. But I think you can make it grammatically nice (with a capital letter) if you had bracketed things.